po/extra/sv.po

```diff
@@ -22,7 +22,7 @@ msgstr ""
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:3
 #: data/com.github.gijsgoudzwaard.image-optimizer.desktop.in:4
 msgid "Image Optimizer"
-msgstr "Bildoptimerare"
+msgstr ""
```

Comment:
Instead of `msgstr ""` it should be `msgstr "Image Optimizer"` otherwise the whole app name will be removed.